main.js

```diff
@@ -1,22 +1,20 @@
 //* Added: rematchRequest onclick, html modal*/
 
-
-
 //Function immediately called
 //alert(message) jQuery display alert box with message
 (function init() {
-  const P1 = 'X';
-  const P2 = 'O';
-  let player;
-  let game;
-  let timer;
-  var makeMoveText = '';
-  var followMoveText = '';
-  var actualFollowMoveText ='';
-  var oppWinStatus = '';
-  const socket = io.connect('/game');
-/////////////////Initialize document//////////////////////
-  $(document).ready(function() {
+	const P1 = 'X';
+	const P2 = 'O';
```

Comment:
Get these out plz

Reply:
u do it

Reply:
also plz pull next time cuz me have to solve conflict :)